src/app.ts

```diff
@@ -0,0 +1,102 @@
+import * as express from 'express'
+import * as compression from 'compression'
+import * as bodyParser from 'body-parser'
+import * as cors from 'cors'
+
+import { config } from './config'
+// import { RequesterFactory } from './bl/requester'
+
+import { Register } from './routes/user/register'
+import { User } from './routes/user/user'
+import { Board } from './routes/board/board'
+import { Card } from './routes/card/card'
+import { Task } from './routes/task/task'
+import { Attachement } from './routes/attachement/attachement'
+import { TaskList } from './routes/taskList/taskList'
+import { List } from './routes/list/list'
+
+export const app = express()
+
+app.set('port', config.server.port)
+app.use(compression())
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+/*
+app.use('*', (req, res, next) => {
```

Comment:
Is this necessary, all the commented part ?

Reply:
Yes, because this code is down. It will be fixed in my next PR.